src/store/reducers/login-reducer.js

```diff
@@ -0,0 +1,23 @@
+const initialState = {
+    email: '',
+    isAuth: false,
+    password: ''
+};
+
+const loginReducer = (state = initialState, action) => {
+    switch (action.type) {
+        case 'SET_USER_DATA':
```

Comment:
move it to the actionTypes

Reply:
https://github.com/mtalstykh-exadel/team4_frontend/blob/27f79d12b2ded8e5b31bd2b44a4a2d0ffd8adfa8/src/components/profile/profile.jsx#L28-L31